js/components/controlBar.js

```diff
@@ -858,149 +782,103 @@ var ControlBar = React.createClass({
-                {...this.props}
-              />
-            </Popover>
+              <Popover
+                popoverClassName="oo-popover oo-popover-pull-right oo-cc-ma-container"
+                autoFocus={this.props.controller.state.multiAudioOptions.autoFocus}
+                closeActionEnabled={this.props.controller.state.accessibilityControlsEnabled}
+                closeAction={this.closePopover.bind(this, CONSTANTS.MENU_OPTIONS.MULTI_AUDIO)}>
+                <ClosedCaptionMultiAudioMenu
+                  language={this.props.language}
+                  localizableStrings={this.props.localizableStrings}
+                  menuClassName={"oo-cc-ma-menu--popover"}
+                  togglePopoverAction={this.closePopover.bind(this, CONSTANTS.MENU_OPTIONS.MULTI_AUDIO)}
+                  {...this.props} />
+              </Popover>
             }
           </div>
         );
-      }.bind(this),
-
-      share: function(alignment) {
-        return (
-          <a
-            className="oo-share oo-control-bar-item"
-            onClick={this.handleShareClick}
-            key="share"
-            aria-hidden="true"
-          >
-            <Icon
-              {...this.props}
-              icon="share"
-              style={dynamicStyles.iconCharacter}
-              onMouseOver={this.highlight}
-              onMouseOut={this.removeHighlight}
-            />
-            <Tooltip
-              enabled={isTooltipEnabled}
-              text={Utils.getLocalizedString(
-                this.props.language,
-                CONSTANTS.SKIN_TEXT.SHARE,
-                this.props.localizableStrings
-              )}
-              responsivenessMultiplier={this.responsiveUIMultiple}
-              bottom={this.responsiveUIMultiple * this.props.skinConfig.controlBar.height}
-              alignment={alignment}
-            />
-          </a>
-        );
-      }.bind(this),
-
-      stereoscopic: function(alignment) {
-        var checkStereoBtn = this.vr && this.isMobile;
-        return !checkStereoBtn ? null : (
-          <AccessibleButton
-            className="oo-video-type oo-control-bar-item oo-vr-stereo-button"
-            onClick={this.handleStereoVrClick}
+      }.bind(this)(),
+
+      share: (
+        <a
+          key={CONSTANTS.CONTROL_BAR_KEYS.SHARE}
+          className="oo-share oo-control-bar-item"
+          onClick={this.handleShareClick}
+          aria-hidden="true">
+          <Icon
+            {...this.props}
+            icon="share"
+            style={dynamicStyles.iconCharacter}
             onMouseOver={this.highlight}
-            onMouseOut={this.removeHighlight}
-            key="stereo"
-            focusId={CONSTANTS.FOCUS_IDS.STEREO}
-            ariaLabel={stereoAriaLabel}
-          >
-            <Icon {...this.props} icon={stereoIcon} style={dynamicStyles.iconCharacter} />
-            <Tooltip
```

Comment:
What happened with the tooltip for stereoscopic? I don't see it in the new changes.

Reply:
I removed it because it had an empty tooltip with no text. I double-checked and the `Stereo` button only shows up on mobile (where tooltips are disabled), so that's probably why no one had noticed before.